doc/release-notes.md.in

```diff
@@ -38,40 +38,56 @@ For detailed information about features and bug fixes in the current release, se
+|------------------------------|----------------|
+| [ESP32](https://www.espressif.com/sites/default/files/documentation/esp32_datasheet_en.pdf) | ✅ |
+| [ESP32c3](https://www.espressif.com/sites/default/files/documentation/esp32-c3_datasheet_en.pdf) | ✅ |
+| [ESP32s2](https://www.espressif.com/sites/default/files/documentation/esp32-s2_datasheet_en.pdf) | ✅ |
+| [ESP32s3](https://www.espressif.com/sites/default/files/documentation/esp32-s3_datasheet_en.pdf) | ✅ |
+
 AtomVM currently supports the following versions of ESP-IDF:
 
-| Espressif supported versions | AtomVM support |
+| IDF SDK supported versions | AtomVM support |
 |------------------------------|----------------|
-| ESP-IDF v4.4 | ✅ |
-| ESP-IDF v5.0 | ✅ |
-| ESP-IDF v5.1 | ✅ |
+| ESP-IDF [v4.4](https://docs.espressif.com/projects/esp-idf/en/v4.4.6/esp32/get-started/index.html) | ✅ |
+| ESP-IDF [v5.0](https://docs.espressif.com/projects/esp-idf/en/v5.0.4/esp32/get-started/index.html) | ✅ |
+| ESP-IDF [v5.1](https://docs.espressif.com/projects/esp-idf/en/v5.1.1/esp32/get-started/index.html) | ✅ |
 
-Building the AtomVM virtual machine for ESP32 is optional.  In most cases, you can simply download a release image from the AtomVM [release](https://github.com/atomvm/AtomVM/releases) repository.  If you wish to work on development of the VM or use one on the additional drivers that are available in the ([AtomVM repositories](https://github.com/atomvm)) you will need a compatible version of ([Espressif's](https:/espressif.com)) ESP-IDF.  Espressif provides excellent [installation documentation](https://docs.espressif.com/projects/esp-idf/en/release-v4.4/esp32/get-started/index.html).
+Building the AtomVM virtual machine for ESP32 is optional.  In most cases, you can simply download a release image from the AtomVM [release](https://github.com/atomvm/AtomVM/releases) repository.  If you wish to work on development of the VM or use one on the additional drivers that are available in the [AtomVM repositories](https://github.com/atomvm) you will to build AtomVM from source.  See the [Build Instructions](build-instructions.md) for information about how to build AtomVM from source code.
 
 ### STM32 Support
 
-The following software is required to build AtomVM for the STM32 platform:
+AtomVM supports deployment on the [STMicroelectronics](https://www.st.com) [STM32](https://www.st.com/en/microcontrollers-microprocessors/stm32-32-bit-arm-cortex-mcus.html) architecture.
+
 
-| Package |
-|---------|
-| [11.3 ARM toolchain](https://developer.arm.com/-/media/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi.tar.xz) |
-| [libopencm3](https://github.com/libopencm3/libopencm3.git) version 0.8.0 |
+AtomVM has been tested on the following development boards:
+
+| STM32 Development Boards | AtomVM support |
+|------------------------------|----------------|
+| [Nucleo-F429ZI](https://www.st.com/en/evaluation-tools/nucleo-f429zi.html) | ✅ |
+
```

Comment:
I guess we can put the [STM32F4Discovery](https://www.st.com/en/evaluation-tools/stm32f4discovery.html) since that is the default target at the moment. CI tests for this chipset and all of the hard coded peripherals (UART related support) are configured for that board, it only makes sense to include it.

Reply:
Added STM32F4Discovery